.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.3/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
+}
```

Comment:
⚠️ Didn't look much into them, so please advise for changes.

Reply:
@smeijer I was considering adding `privatePackages: { version: true, tag: true }` but I have to admit I don't fully understand its effects, even after reading the docs: https://github.com/changesets/changesets/blob/main/docs/versioning-apps.md 😊

Reply:
Not entirely sure either, I think things look good like this. Let's just try it, and fix later if it's not?